wagn/lib/wagn/generators/wagn/templates/config/application.rb

```diff
@@ -43,7 +43,8 @@ class Application < Wagn::Application
     # config.paths['files'] = 'files'
     # where uploaded files are actually stored. (eg Image and File cards)
 
-    # config.paths['local-mod'] = 'mod'
+    # This probably needs to be a full path or it will be gem relative, which isn't really what we want here
+    # config.paths['mods'] << 'mod'
```

Comment:
let's get rid of this comment (a template is a bad place for a code comment).

Also, I think it's incorrect.  by default it will be handled as deck relative (all gem paths must be made absolute via add_gem_path)